manifests/charts/base/templates/default.yaml

```diff
@@ -23,6 +23,9 @@ webhooks:
         namespace: {{ .Values.global.istioNamespace }}
         path: "/validate"
       {{- end }}
+      {{- if .Values.base.validationCABundle }}
+      caBundle: "{{ .Values.base.validationCABundle }}"
```

Comment:
In some case we donot know CABundle before we run istiod up

Reply:
right which is why this is a opt in - folks can set this value if they know ahead of time